content/news/XXXX-XX-XX-mixxx-2-3-0-released.md

```diff
@@ -0,0 +1,122 @@
+
+2.3.0 also adds out-of-the-box support for the [Pioneer DDJ-200](https://manual.mixxx.org/2.3/en/hardware/controllers/pioneer_ddj_200.html) and [DDJ-400](https://manual.mixxx.org/2.3/en/hardware/controllers/pioneer_ddj_400.html), the [Native Instruments Traktor Kontrol S3](https://manual.mixxx.org/2.3/en/hardware/controllers/native_instruments_traktor_kontrol_s3.html), the [Hercules DJControl Inpulse 200](https://manual.mixxx.org/2.3/en/hardware/controllers/hercules_djcontrol_inpulse_200.html) and [Jogvision](https://manual.mixxx.org/2.3/en/hardware/controllers/hercules_djcontrol_jogvision.html), the [Roland DJ-505](https://manual.mixxx.org/2.3/en/hardware/controllers/roland_dj_505.html), the [Behringer B-Control BCR2000](https://manual.mixxx.org/2.3/en/hardware/controllers/behringer_bcr2000.html) and [DDM4000](https://manual.mixxx.org/2.3/en/hardware/controllers/behringer_ddm4000.html), the [ION Discover DJ Pro](https://manual.mixxx.org/2.3/en/hardware/controllers/ion_discover_dj_pro.html) and the [Numark iDJ Live II](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_idj_live_ii.html). A few existing controller mappings have received fixes and new features, too.
+
+For the full list of changes, have a look at the [changelog](https://manual.mixxx.org/2.3/en/chapters/appendix/version_history.html) and the [2.3.0 milestone on Launchpad](https://launchpad.net/mixxx/+milestone/2.3.0).
+
+This has been a rather big release, with lots of useful changes, bug fixes and improvements.
+All in all, a total of 7183 changes changed over 1 million lines of code were changed since the [2.2.4 release]({filename}/news/2020-06-11-Mixxx-2-2-4-released.md)!
+
+**TODO:** Update the numbers above using `git shortlog -sn release-2.2.4..upstream/2.3 | wc -l`, `git log --oneline --no-merges release-2.2.4..upstream/2.3 -- . ':!lib' | wc -l` and `git diff --shortstat release-2.2.4..upstream/2.3 -- . ':!lib`.
+
+Many people spent their free time working on Mixxx and reported bugs, translated Mixxx into other languages, improved the manual, contributed controller mappings, improved skins and hacked on the core code. Thanks a lot to all of them, we really appreciate it!
+
+The Mixxx code (including skins and controller mappings) has received contributions by almost 100 people:
+
+    $ git shortlog -sn release-2.2.4..upstream/2.3
+    2305  Jan Holthuis
+    1838  Uwe Klotz
+    1393  Daniel Schürmann
+    1100  ronso0
+     899  Be
+     264  Owen Williams
+     144  Nino Miškić-Pletenac
+     131  Ferran Pujol Camins
+     118  RJ Skerry-Ryan
+      81  Evan Dekker
+      79  Harshit Maurya
+      71  Swiftb0y
+      65  JoergAtGithub
+      63  S.Brandt
+      57  Kerrick Staley
+      52  Daniel Poelzleithner
+      52  Matthew Nicholson
+      50  Justin Kourie
+      49  Christian
+      36  JosepMaJAZ
```

Comment:
josep is here under multiple names.  We need to dedupe

Reply:
I don't know. I thought by just adding the shortlog, we'd have a reproducible, simple way to show the list of contributors. I think it's okay like this. The alternative would be to take the contributor list from the about dialog.

Reply:
The shortlog is fine I guess, but maybe remove the commit counts?

Reply:
Done.